functions/index.js

```diff
@@ -58,16 +69,16 @@ exports.getReport = functions.https.onRequest((req, res) => {
   });
 });
 
+
 /**
  * Internal helper method to build database queries given some parameters.
  * Currently accepts species, neighborhood, season, year, and time_of_day as fields
  * that are acceptable to search. Always filters by at least one week old.
  */
 buildQuery = (queryParams, collection) => {
   // always filter by time_submitted
-  let week_ago = moment().subtract(1, 'week').toISOString();
-  //let initialQuery = collection.where('timestamp', '<=', week_ago);
-  let initialQuery = collection;
+  let week_ago = toTimestamp(moment().subtract(1, 'week').toDate());
+  let initialQuery = collection.where('time_submitted', '<=', week_ago);
```

Comment:
The form updates the db field -> "timestamp". We should either change that to time_submitted or use timestamp here.

Reply:
The addReports endpoint automatically adds a ‘time_submitted’ field with the current time stamp now. The two should be distinct because a user might report a picture they took weeks ago but we still want to have a week to review their submission.

Reply:
duh! I remember it now!